Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/specflow.json

```diff
@@ -1,10 +1,11 @@
 ﻿{
   "stepAssemblies": [
     { "assembly": "Aquality.Selenium.Template.SpecFlow" },
-    { "assembly": "Allure.SpecFlowPlugin" }
+    { "assembly": "Allure.SpecFlowPlugin" },
+    { "assembly": "AqualityTracking.SpecFlowPlugin" }
   ],
   "runtime": {
     "stopAtFirstError": true,
     "missingOrPendingStepsOutcome": "Error"
   }
```

Comment:
Don't we need section "plugins"
`
  "plugins": [
    {
      "name": "Allure",
      "type": "Runtime"
    }
  ],`
here?

Reply:
No, it is not necessary, only assembly.